### PR TITLE
Update the TomEE content.md File

### PR DESCRIPTION
**Pull request by gerdogdu**
Hi all
As indicate in here http://tomee.apache.org/commercial-support.html , ManageCat is another commerical support company behind the Apache TomEE project. We would also like to include ManageCat to this README file in addition to the another commercial company Tomitribe for supporting the project.

Thanks 
Gurkan

### CODE REVIEW

**Comment by yosifkit** [comment]
CI is failing: https://travis-ci.org/docker-library/docs/builds/168410637#L214-L227.

**Comment by gerdogdu** [comment]
cc @tianon 
I really don't understanf why markdown format is failing. It does not give error in my local side.

@lordofthejars Can you help me to update this file correctly?

Thanks
Gurkan-

File: tomee/content.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ Below are some press releases that might be helpful:
 -	Tomitribe Announces Enterprise Service Support for Apache TomEE http://www.tomitribe.com/company/press/tomitribe_enterprise_service_support_for_apache_tomee_javaone_2013/
 -	Tomitribe Introduces Community Partnership Program Conference http://www.tomitribe.com/company/press/tomitribe-introduces-community-partnership-program-and-presents-java-ee-sessions-at-javaone-2014-conference/
 
+# Apache TomEE and ManageCat
+
+Another commercial support company provides enterprise support for Apache TomEE is ManageCat. ManageCat involves with a lot of Apache Java EE projects to contribute open source Java EE ecosystem. ManageCat co-founder is also a founder of open source Apache OpenWebBeans project (http://openwebbans.apache.org) which is heavily used in Apache TomEE application server for CDI integration.
```
Comment (yosifkit) [inline tomee/content.md]: The CI is failing on the extra space at the end of this line.  You have to run `./markdownfmt.sh -d` to see the diff of changes that markdownfmt requires.

**Comment by gerdogdu** [comment]
Thanks I updated and now fixed.

**Comment by lordofthejars** [comment]
@gerdogdu I am not from Docker Inc, so I cannot merge

**Comment by gerdogdu** [comment]
Hi @tianon  and @yosifkit 
@lordofthejars  is ok with the change but as you see his comment, he is not authorized to merge. Can you help to merge?

Thanks.
Gurkan-

**Pull request merged**